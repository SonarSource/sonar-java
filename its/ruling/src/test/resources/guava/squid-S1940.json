{
'com.google.guava:guava:src/com/google/common/math/MathPreconditions.java':[
73,
],
}
