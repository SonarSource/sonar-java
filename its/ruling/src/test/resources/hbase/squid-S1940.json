{
'org.apache.hbase:hbase-protocol:src/main/java/org/apache/hadoop/hbase/protobuf/generated/AccessControlProtos.java':[
1408,
1425,
2121,
2289,
2306,
2738,
2891,
2908,
3259,
4085,
4229,
4712,
5309,
8183,
8577,
8904,
9298,
],
'org.apache.hbase:hbase-protocol:src/main/java/org/apache/hadoop/hbase/protobuf/generated/AdminProtos.java':[
1614,
2161,
2316,
2672,
3191,
3585,
3948,
4147,
4864,
5608,
6027,
6044,
6486,
12589,
12745,
13332,
13924,
16083,
16701,
16959,
17360,
18370,
18736,
],
'org.apache.hbase:hbase-protocol:src/main/java/org/apache/hadoop/hbase/protobuf/generated/AggregateProtos.java':[
1040,
1465,
],
'org.apache.hbase:hbase-protocol:src/main/java/org/apache/hadoop/hbase/protobuf/generated/ClientProtos.java':[
178,
539,
1256,
1698,
2003,
2011,
3058,
3298,
4195,
4843,
7969,
7982,
8472,
9755,
10877,
11150,
13749,
13757,
15049,
15289,
17972,
17982,
18008,
18021,
18031,
19011,
19247,
19627,
20183,
21441,
26424,
27115,
29515,
30011,
30524,
31059,
31554,
32006,
],
'org.apache.hbase:hbase-protocol:src/main/java/org/apache/hadoop/hbase/protobuf/generated/ClusterStatusProtos.java':[
2522,
3008,
3762,
6001,
8168,
8176,
8199,
9547,
9859,
10400,
11922,
11930,
11938,
11959,
11980,
13176,
13416,
13656,
14013,
14370,
],
'org.apache.hbase:hbase-protocol:src/main/java/org/apache/hadoop/hbase/protobuf/generated/ErrorHandlingProtos.java':[
1095,
1856,
],
'org.apache.hbase:hbase-protocol:src/main/java/org/apache/hadoop/hbase/protobuf/generated/FilterProtos.java':[
5006,
5571,
6786,
7134,
7294,
7688,
8883,
9231,
13979,
13989,
14350,
16674,
17056,
],
'org.apache.hbase:hbase-protocol:src/main/java/org/apache/hadoop/hbase/protobuf/generated/HBaseProtos.java':[
988,
996,
1004,
1807,
2047,
2287,
3824,
3832,
4428,
4668,
6125,
6524,
13085,
13932,
16481,
16964,
],
'org.apache.hbase:hbase-protocol:src/main/java/org/apache/hadoop/hbase/protobuf/generated/HFileProtos.java':[
95,
493,
],
'org.apache.hbase:hbase-protocol:src/main/java/org/apache/hadoop/hbase/protobuf/generated/MapReduceProtos.java':[
91,
473,
823,
1346,
],
'org.apache.hbase:hbase-protocol:src/main/java/org/apache/hadoop/hbase/protobuf/generated/MasterProcedureProtos.java':[
1360,
2137,
3649,
4691,
5072,
5849,
11523,
11531,
12394,
12634,
],
'org.apache.hbase:hbase-protocol:src/main/java/org/apache/hadoop/hbase/protobuf/generated/MasterProtos.java':[
9014,
9649,
20560,
20954,
21798,
22192,
23036,
23430,
31404,
31798,
37253,
37849,
38357,
38751,
39842,
40236,
50793,
50810,
51270,
],
'org.apache.hbase:hbase-protocol:src/main/java/org/apache/hadoop/hbase/protobuf/generated/MultiRowMutationProtos.java':[
787,
1278,
],
'org.apache.hbase:hbase-protocol:src/main/java/org/apache/hadoop/hbase/protobuf/generated/ProcedureProtos.java':[
507,
517,
1993,
3026,
3036,
3392,
4842,
4989,
4999,
5010,
5020,
5519,
5585,
6004,
6362,
7010,
],
'org.apache.hbase:hbase-protocol:src/main/java/org/apache/hadoop/hbase/protobuf/generated/RegionServerStatusProtos.java':[
1073,
1502,
4827,
5351,
5809,
6546,
7033,
7668,
],
'org.apache.hbase:hbase-protocol:src/main/java/org/apache/hadoop/hbase/protobuf/generated/SecureBulkLoadProtos.java':[
130,
736,
],
'org.apache.hbase:hbase-protocol:src/main/java/org/apache/hadoop/hbase/protobuf/generated/SnapshotProtos.java':[
1161,
2157,
2640,
3500,
3854,
4441,
],
'org.apache.hbase:hbase-protocol:src/main/java/org/apache/hadoop/hbase/protobuf/generated/VisibilityLabelsProtos.java':[
91,
485,
1347,
1741,
2072,
2509,
2673,
2683,
3123,
3277,
3671,
4447,
4884,
5542,
5890,
],
'org.apache.hbase:hbase-protocol:src/main/java/org/apache/hadoop/hbase/protobuf/generated/WALProtos.java':[
1332,
1345,
2688,
2961,
4184,
4192,
5078,
5207,
5627,
5927,
6577,
7450,
7887,
8537,
8811,
9544,
10019,
10985,
],
'org.apache.hbase:hbase-protocol:src/main/java/org/apache/hadoop/hbase/protobuf/generated/ZooKeeperProtos.java':[
4115,
4123,
4975,
5215,
],
}
