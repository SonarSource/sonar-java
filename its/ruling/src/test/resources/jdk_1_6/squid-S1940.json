{
'jdk_1_6:java/awt/geom/CubicCurve2D.java':[
1227,
],
'jdk_1_6:java/beans/ReflectionUtils.java':[
298,
308,
],
'jdk_1_6:java/net/DatagramSocket.java':[
854,
],
'jdk_1_6:java/net/PlainSocketImpl.java':[
259,
],
'jdk_1_6:java/net/ServerSocket.java':[
748,
],
'jdk_1_6:java/net/Socket.java':[
1065,
],
'jdk_1_6:java/nio/channels/Channels.java':[
217,
],
'jdk_1_6:java/sql/Date.java':[
117,
],
'jdk_1_6:java/text/ChoiceFormat.java':[
369,
],
'jdk_1_6:java/util/concurrent/ConcurrentHashMap.java':[
586,
],
}
