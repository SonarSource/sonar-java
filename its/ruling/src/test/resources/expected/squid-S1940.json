{
'project:com/sun/corba/se/impl/util/PackagePrefixChecker.java':[
52,
],
'project:com/sun/org/apache/regexp/internal/RE.java':[
1030,
1038,
1046,
],
'project:com/sun/org/apache/xerces/internal/impl/XMLEntityScanner.java':[
1701,
],
'project:com/sun/org/apache/xerces/internal/impl/dv/xs/XSSimpleTypeDecl.java':[
1486,
1486,
1493,
1493,
1500,
1500,
1507,
1507,
1514,
1521,
],
'project:com/sun/org/apache/xml/internal/dtm/ref/DTMDefaultBaseTraversers.java':[
392,
609,
],
'project:com/sun/org/apache/xpath/internal/SourceTreeManager.java':[
178,
],
'project:java/awt/geom/CubicCurve2D.java':[
1405,
],
'project:java/net/AbstractPlainSocketImpl.java':[
246,
],
'project:java/net/DatagramSocket.java':[
909,
],
'project:java/net/ServerSocket.java':[
808,
],
'project:java/net/Socket.java':[
1141,
],
'project:java/nio/channels/Channels.java':[
381,
],
'project:java/text/ChoiceFormat.java':[
386,
],
'project:java/util/concurrent/ConcurrentHashMap.java':[
737,
],
'project:javax/sound/sampled/DataLine.java':[
459,
],
'project:javax/sound/sampled/Port.java':[
174,
],
'project:javax/sql/rowset/RowSetMetaDataImpl.java':[
140,
],
'project:javax/swing/JComponent.java':[
3094,
3104,
],
'project:javax/swing/JTable.java':[
5194,
],
'project:javax/swing/plaf/basic/BasicPopupMenuUI.java':[
844,
],
}
